integration-tests/sanity-functional-protractor/specs/api-spec.js

```diff
@@ -0,0 +1,121 @@
+'use strict';
+
+//npm test -- --specs specs/api-spec.js
+
+const PublishPage = require('../page_objects/abstract-po.js');
```

Comment:
Why some of the consts are capitalized and other don't? I don't see any rule here.

Reply:
cleaned up